### PR TITLE
Add weekly update 2015-11-27

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by mikeal** [comment]
:heartbeat:

File: locale/en/blog/weekly-updates/weekly-update.2015-11-27.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### Notable changes
+
+* [[`7d11208f68`](https://github.com/nodejs/node/commit/7d11208f68)] - **build**: backport tools/release.sh (Rod Vagg) [#3642](https://github.com/nodejs/node/pull/3642)
+* [[`6fb0b92fa0`](https://github.com/nodejs/node/commit/6fb0b92fa0)] - **build**: backport config for new CI infrastructure (Rod Vagg) [#3642](https://github.com/nodejs/node/pull/3642)
+* [[`83441616a5`](https://github.com/nodejs/node/commit/83441616a5)] - **build**: fix --without-ssl compile time error (Ben Noordhuis) [#3825](https://github.com/nodejs/node/pull/3825)
+
+### CVE-2015-8027 Denial of Service Vulnerability / CVE-2015-6764 V8 Out-of-bounds Access Vulnerability
+
+* Security release coming on December 1 or December 2 (depending on your timezone) will resolve a denail-of-service vulnerability. Be prepared to update!
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2015-11-27.md]: Typo: denail -> denial

File: locale/en/blog/weekly-updates/weekly-update.2015-11-27.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* UTC will be made available with appropriate fixes for CVE-2015-8027 and CVE-2015-6764 (for v4.x and v5.x only)
```
Comment (Trott) [inline locale/en/blog/weekly-updates/weekly-update.2015-11-27.md]: UTC? ??

**Comment by Trott** [comment]
Two small comments, but otherwise looks good to me.

**Comment by JungMinu** [comment]
@mikeal @Trott Thanks for your comments, updated :smile:

**Comment by Trott** [comment]
:+1:

**Pull request merged**